apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -81,7 +95,8 @@ const LoginTypeSelection: React.FunctionComponent = () => {
           sourceParam,
           PLATFORMS.BOTH
         );
-        sessionStorage.setItem(ACCOUNT_TYPE_SESSION_KEY, userType as string);
+        sessionStorage.setItem(ACCOUNT_TYPE_SESSION_KEY, urlUserType as string);
+        setUserType(getUserType());
```

Comment:
```suggestion
        setUserType(urlUserType);
```
I think you can skip some function calls with this change

Reply:
Whoops that's definitely what I meant, thanks for catching this!